packages/core/src/integrations/inboundfilters.ts

```diff
@@ -28,42 +29,21 @@ export interface InboundFiltersOptions {
   disableTransactionDefaults: boolean;
 }
 
-/** Inbound filters configurable by the user */
-export class InboundFilters implements Integration {
-  /**
-   * @inheritDoc
-   */
-  public static id: string = 'InboundFilters';
-
-  /**
-   * @inheritDoc
-   */
-  public name: string;
-
-  private readonly _options: Partial<InboundFiltersOptions>;
-
-  public constructor(options: Partial<InboundFiltersOptions> = {}) {
-    this.name = InboundFilters.id;
-    this._options = options;
-  }
-
-  /**
-   * @inheritDoc
-   */
-  public setupOnce(_addGlobalEventProcessor: unknown, _getCurrentHub: unknown): void {
-    // noop
+const inboundFiltersIntegration = makeIntegrationFn('InboundFilters', (options: Partial<InboundFiltersOptions>) => {
```

Comment:
I just converted a single integration for now to show this in "reality", but I can also extract this out into a follow up PR.

Reply:
Let's just keep it here, the change is pretty small.